src/node_http_parser.cc

```diff
@@ -478,7 +460,144 @@ class Parser : public BaseObject {
   }
 
 
- private:
+  static void Consume(const FunctionCallbackInfo<Value>& args) {
+    Parser* parser = Unwrap<Parser>(args.Holder());
+    Local<External> stream_obj = args[0].As<External>();
```

Comment:
@indutny Did anything need to happen here? Just asking to be sure nothing was overlooked.